demo-project/src/demo_project/hooks.py

```diff
@@ -11,7 +11,7 @@ class TimingHooks:
     """
     This class is designed to highlight how hooks can be added
     to the lifecycle of a Kedro run. You can read more here:
-    https://kedro.readthedocs.io/en/latest/07_extend_kedro/02_hooks.html
+    https://kedro.readthedocs.io/en/stable/07_extend_kedro/02_hooks.html
```

Comment:
This page doesn't exist anymore actually, let's use https://docs.kedro.org/en/stable/hooks/ instead